src/main/java/io/stargate/sgv2/jsonapi/api/model/command/impl/CreateCollectionCommand.java

```diff
@@ -41,12 +41,12 @@ public record Options(
           VectorizeConfig vectorize) {
 
     public record VectorSearchConfig(
-        @Positive(message = "size should be greater than `0`")
+        @Positive(message = "dimension should be greater than `0`")
             @Schema(
-                description = "Vector field embedding size",
+                description = "Dimension of the vector field",
                 type = SchemaType.INTEGER,
                 implementation = Integer.class)
-            Integer size,
+            Integer dimension,
```

Comment:
I think combination of:

```
@JsonProperty("dimension")
@JsonAlias("size")
```

should work here, to support old name too (actually `@JsonProperty` is optional mostly for documentation, only alias needed)